examples/cfd/09_Darcy_flow_equation.ipynb

```diff
@@ -0,0 +1,436 @@
+{
```

Comment:
<p>Maybe show a few different samples to show what this generator does?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1998/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>